src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -2376,30 +2406,14 @@ void FixedwingPositionControl::control_backtransition(const float control_interv
 		_lpos_where_backtrans_started = curr_pos_local;
 	}
 
-	float roll_body{0.0f};
-
-	if (_control_mode.flag_control_position_enabled) {
-		navigateLine(_lpos_where_backtrans_started, curr_wp_local, curr_pos_local, ground_speed, _wind_vel);
-		roll_body = getCorrectedNpfgRollSetpoint();
-	}
-
-	target_airspeed = _npfg.getAirspeedRef() / _eas2tas;
+	navigateLine(_lpos_where_backtrans_started, curr_wp_local, curr_pos_local, ground_speed, _wind_vel);
+	const float roll_body = getCorrectedNpfgRollSetpoint();
 
-	float yaw_body = _yaw; // yaw is not controlled, so set setpoint to current yaw
-
-	tecs_update_pitch_throttle(control_interval,
```

Comment:
Did I get that correctly: we don't need the TECS call here because pitch and thrust are anyway overwritten by the transition controller in the VTOL module?

Reply:
@sfuhrer Exactly